app/components/at_mention/at_mention.test.js

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import {Clipboard, Platform} from 'react-native';
+
+import mattermostManaged from 'app/mattermost_managed';
+
+import Preferences from 'mattermost-redux/constants/preferences';
+
+import AtMention from './at_mention';
+
+jest.mock('react-intl');
+
+describe('AtMention', () => {
+    const baseProps = {
+        isSearchResult: false,
+        mentionName: 'user-1',
+        mentionStyle: {color: '#2389d7'},
+        navigator: {push: jest.fn(), showModal: jest.fn()},
+        onLongPress: jest.fn(),
+        onPostPress: jest.fn(),
+        textStyle: {color: '#3d3c40', fontSize: 15, lineHeight: 20},
+        teammateNameDisplay: 'username',
+        theme: Preferences.THEMES.default,
+        usersByUsername: {'user-1': {id: 'user-1', username: 'username_1'}},
+    };
+
+    test('should match snapshot', () => {
+        const wrapper = shallow(
+            <AtMention {...baseProps}/>
+        );
+
+        expect(wrapper.getElement()).toMatchSnapshot();
+    });
+
+    test('should match snapshot, for user without username', () => {
+        const usersByUsername = {'user-1': {id: 'user-1'}};
+        const wrapper = shallow(
+            <AtMention
+                {...baseProps}
+                usersByUsername={usersByUsername}
+            />
+        );
+
+        expect(wrapper.getElement()).toMatchSnapshot();
+    });
+
+    test('should match snapshot, with suffix', () => {
```

Comment:
What is this testing? Name not descriptive.

Reply:
Changed description to "should match text if mentioned name is with trailing punctuation"